docs/vsphere_ci/README.md

```diff
@@ -37,9 +37,8 @@ The `scripts/autounattend.xml` file must be edited to change the value of `Windo
 
 The provided [autounattend.xml](scripts/autounattend.xml)
 - Installs VMWare tools
-- Runs powershell.ps1 script which configures SSH. This is different from [powershell script](../powershell.ps1) we have 
-earlier as it copies the file from floppy disk instead of copying from C drive
```

Comment:
Found the `powershell.ps1`  script with duplicated content, except for the [line that copies](https://github.com/jrvaldes/windows-machine-config-operator/blob/f1158e08355613f9a75c61c11ed6208c28f98a34/docs/vsphere_ci/scripts/powershell.ps1#L21-L21) the `authorized_keys` file. 

In order to consolidate them, the novel `install-openssh.ps1` script accept the path for the `authorized_keys` file as parameter.  See usage in the [autounattend.xml files](https://github.com/jrvaldes/windows-machine-config-operator/blob/6af27634325b58c29668a1d9bf29084bd7f3e953/docs/vsphere_ci/scripts/autounattend.xml#L106-L106)